### ISSUE TITLE
Feature request: Set date format separately from language preference

### ISSUE THREAD

**Issue by digicommons** [almarklein/timetagger#381]
After using timetagger on an OS entirely in German, I am now on Ubuntu set to English (United States) but with a format prefererence of Germany (03.07.2023, 12:34:56, 123.456,00 etc.). On timetagger the date input now defaults to 07/03/2023, instead of the desired 03.07.2023.

Is it possible to access a user's date format preferences separately from language through the web/browser API? Or would it be possible to set a date format preference in timetagger's settings?

Thank you.

**Comment by almarklein** [comment]
Interesting question!

The dialogs simply uses `<input type='date'>`, e.g.:
https://github.com/almarklein/timetagger/blob/e438cad56e10384ccbf2678f8a3dcaa216baa7f4/timetagger/app/dialogs.py#L668

So it will use the format whatever the browser thinks is appropriate. This is *usually* the best option, but in your cases the browser seems to have it wrong.

**Comment by digicommons** [comment]
You are right, `<input type='date'>` is definitely the most appropriate and versatile option here.

Though, as this was bugging me, I found a [good explanation](https://bugzilla.mozilla.org/show_bug.cgi?id=1464592#c5) of why this might be happening in my case (Ubuntu+Firefox/Chromium). It seems that Windows works around this by allowing users to set a locale of `English (Germany)`. Setting the locale to `en_IE.UTF-8` as described [here](https://unix.stackexchange.com/a/62317), unfortunately didn't change anything.

Closing issue and thank you anyways for your quick replies to issues!
